test-data/expected-result-data/ssh-result.json

```diff
@@ -0,0 +1,807 @@
+{
+  "datasetVersion": {
+    "datasetPersistentId": "doi:10.5072/TSS/SYDZNZ",
```

Comment:
Volgens mij moet deze leeg zijn. De test breekt nu omdat de expected result een DOI heeft maar in je test voeg je de DOI pas na de assertion toe.
Dit zou dus `"datasetPersistentId": ""` moeten zijn.